test/arguments_test.cpp

```diff
@@ -158,3 +158,95 @@ TEST(ArgumentsTest, IpAddrMask)
     ASSERT_THROW(args.asIpAddrMask(), std::invalid_argument);
     ASSERT_THROW(args.asIpAddrMask(), std::invalid_argument);
 }
+
+TEST(ArgumentsTest, RemoteAddressPositive)
+{
+    char* testArgs[] = {
+        const_cast<char*>("127.0.0.1"),
```

Comment:
It is not a remote address =)

Reply:
It's just test that the IP is valid value.